install_config/registry/accessing_registry.adoc

```diff
@@ -311,3 +311,95 @@ configuration].
 Refer to
 link:https://prometheus.io/docs/querying/basics/[upstream documentation] for
 more advanced queries and recommended visualizers.
+
+[[accessing-registry-image-signatures]]
+== Reading and Writing Image Signatures
+
+The OpenShift Container Registry provides an `extensions` endpoint that allows
+you to write and read image signatures. The image signatures are stored in the
+{product-title} key-value store via the Docker Registry API.
+
+[NOTE]
+====
+This endpoint is experimental and not supported by the upstream Docker Registry
+project. See the link:https://docs.docker.com/registry/spec/api/[upstream API
+documentation] for general information about the Docker Registry API.
+====
+
+[[writing-image-signatures]]
+=== Writing Image Signatures
+
+In order to add a new signature to the image, users can use the HTTP `PUT`
+method together with the JSON payload sent the `extensions` endpoint:
+
+----
+PUT /extensions/v2/<namespace>/<name>/signatures/<digest>
+----
+
+----
+$ curl -X PUT --data @signature.json http://<user>:<token>@<registry_endpoint>:5000/extensions/v2/<namespace>/<name>/signatures/sha256:<digest>
+----
+
+The JSON payload with the signature content should have the following structure:
+
+----
+{
+  "version": 2,
+  "type":    "atomic",
+  "name":    "sha256:4028782c08eae4a8c9a28bf661c0a8d1c2fc8e19dbaae2b018b21011197e1484@cddeb7006d914716e2728000746a0b23",
+  "content": "<base64_encoded_signature>"
```

Comment:
The original had a comma at the end here and in the other JSON example, which I've removed.